<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>531</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="usesScrollButtons">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Contrato</string>
    </attribute>
    <widget class="QTableView" name="tabelaContrato">
     <property name="geometry">
      <rect>
       <x>-2</x>
       <y>-2</y>
       <width>417</width>
       <height>501</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabWidgetPage1">
    <attribute name="title">
     <string>Casa</string>
    </attribute>
    <widget class="QTableView" name="tabelaCasa">
     <property name="geometry">
      <rect>
       <x>-2</x>
       <y>-2</y>
       <width>421</width>
       <height>501</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabWidgetPage2">
    <attribute name="title">
     <string>Inquilino</string>
    </attribute>
    <widget class="QTableView" name="tabelaInq">
     <property name="geometry">
      <rect>
       <x>-2</x>
       <y>-2</y>
       <width>417</width>
       <height>501</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="Gerar_recibo">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>110</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>gerar recibo</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="Mes">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>110</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumDate">
    <date>
     <year>2016</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::MonthSection</enum>
   </property>
   <property name="displayFormat">
    <string>MM/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>false</bool>
   </property>
   <property name="currentSectionIndex">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>40</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>MÃªs/ano</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
